Fastlane/deployment_lanes.rb

```diff
@@ -255,7 +255,7 @@
         demo_account_password: options[:demo_account_password] || ENV['BETA_DEMO_ACCOUNT_PASSWORD']
       },
       skip_waiting_for_build_processing: false,
-      skip_submission: true,
+      skip_submission: false, # Ensure distributing the release build to TestFlight groups.
```

Comment:
Won't this increase the time for the workflow?

Reply:
I don't expect so!